x/staking/types/msg.go

```diff
@@ -62,10 +69,10 @@ func (msg MsgCreateValidator) GetSigners() []sdk.AccAddress {
 	return addrs
 }
 
-// TODO Remove use of custom struct (no longer necessary)
-// get the bytes for the message signer to sign on
-func (msg MsgCreateValidator) GetSignBytes() []byte {
-	b, err := MsgCdc.MarshalJSON(struct {
+// MarshalJSON implements the json.Marshaler interface to provide custom JSON
+// serialization of the MsgCreateValidator type.
+func (msg MsgCreateValidator) MarshalJSON() ([]byte, error) {
```

Comment:
If we _don't_ care about the pubkey being a string when rendered as JSON (e.g. for clients/UIs), then we don't even need this.

Reply:
I think we do care about the pubkey being marshaled as a string tho right?

Reply:
We do care, e.g. for checking on the Ledger screen.

Reply:
related https://github.com/cosmos/cosmos-sdk/issues/3228

Reply:
Thanks guys -- resolving as nothing to do here.